Lab 1/Lab1/IOSystem/Writers/WriterJSON.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization.Json;
+using System.IO;
+
+namespace Lab1
```

Comment:
Just a reminder about namespaces.
Namespaces should be the same as the folders structure. Fix in all places.